### PR TITLE
Julia v0.6 fix

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by fredrikekre** [comment]
Test segfaultar:

```julia
signal (4): Illegal instruction
while loading /home/fredrik/Dropbox/Programming/JuAFEM.jl/test/test_VTK.jl, in expression starting on line 2
vtk_grid at /home/fredrik/Dropbox/Programming/JuAFEM.jl/src/deprecations.jl:20
macro expansion at /home/fredrik/Dropbox/Programming/JuAFEM.jl/test/test_VTK.jl:64 [inlined]
macro expansion at ./test.jl:853 [inlined]
anonymous at ./<missing> (unknown line)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_toplevel_eval_flex at /home/fredrik/julia-master/src/toplevel.c:589
jl_parse_eval_all at /home/fredrik/julia-master/src/ast.c:874
jl_load at /home/fredrik/julia-master/src/toplevel.c:616
include_from_node1 at ./loading.jl:539
unknown function (ip: 0x7f5af5b8dc52)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
include at ./sysimg.jl:14
unknown function (ip: 0x7f5d070d2f3b)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
do_call at /home/fredrik/julia-master/src/interpreter.c:75
eval at /home/fredrik/julia-master/src/interpreter.c:230
jl_interpret_toplevel_expr at /home/fredrik/julia-master/src/interpreter.c:34
jl_toplevel_eval_flex at /home/fredrik/julia-master/src/toplevel.c:577
jl_parse_eval_all at /home/fredrik/julia-master/src/ast.c:874
jl_load at /home/fredrik/julia-master/src/toplevel.c:616
include_from_node1 at ./loading.jl:539
unknown function (ip: 0x7f5af5b8dc52)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
include at ./sysimg.jl:14
unknown function (ip: 0x7f5d070d2f3b)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
do_call at /home/fredrik/julia-master/src/interpreter.c:75
eval at /home/fredrik/julia-master/src/interpreter.c:230
jl_interpret_toplevel_expr at /home/fredrik/julia-master/src/interpreter.c:34
jl_toplevel_eval_flex at /home/fredrik/julia-master/src/toplevel.c:577
jl_toplevel_eval_flex at /home/fredrik/julia-master/src/toplevel.c:551
jl_toplevel_eval_in at /home/fredrik/julia-master/src/builtins.c:484
eval at ./boot.jl:235
unknown function (ip: 0x7f5d071f097f)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
eval_user_input at ./REPL.jl:66
unknown function (ip: 0x7f5af5b66736)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
macro expansion at ./REPL.jl:97 [inlined]
#1 at ./event.jl:73
unknown function (ip: 0x7f5af5b61f0f)
jl_call_method_internal at /home/fredrik/julia-master/src/julia_internal.h:248 [inlined]
jl_apply_generic at /home/fredrik/julia-master/src/gf.c:2212
jl_apply at /home/fredrik/julia-master/src/julia.h:1410 [inlined]
start_task at /home/fredrik/julia-master/src/task.c:261
unknown function (ip: 0xffffffffffffffff)
Allocations: 46120131 (Pool: 46114149; Big: 5982); GC: 90
Illegal instruction (core dumped)
```

**Comment by KristofferC** [comment]
Ta bort VTK deprecationen?

File: test/test_VTK.jl [hunk 1]
```diff
@@ -46,49 +46,33 @@ using SHA
                                        ("tetras", coords_3D, tetra_mesh_edof, VTKCellTypes.VTK_TETRA))
 
         vtkfile = vtk_grid(name, coord, edof, ctype)
```
Comment (fredrikekre) [inline test/test_VTK.jl]: This line segfaults... But only if this file is ran with `include`, copy pasting the full testset in the REPL passes just fine.

**Comment by KristofferC** [comment]
https://github.com/JuliaLang/julia/issues/17003

File: docs/make.jl [hunk 2]
```diff
@@ -5,7 +5,7 @@ makedocs(
     format = :html,
     sitename = "JuAFEM.jl",
     doctest = true,
-    strict = true,
+    strict = false, # VERSION.minor == 6, # only strict mode on release bot
```
Comment (fredrikekre) [inline docs/make.jl]: Doing this since some packages still spew out deprecations...

File: test/test_VTK.jl [hunk 3]
```diff
@@ -1,5 +1,7 @@
 using SHA
 @testset "VTK" begin
+    if VERSION.minor < 6 # temporarily skip tests on this until
```
Comment (fredrikekre) [inline test/test_VTK.jl]: Skipping 0.6 tests here. Seems to work outside the test suite so thats nice at least.

```julia
julia> function foo()
           grid = generate_grid(Triangle, (2, 2))
           vtk = vtk_grid("testfile", grid)
           vtk_save(vtk)
       end
foo (generic function with 1 method)

julia> foo()
1-element Array{String,1}:
 "testfile.vtu"
```

**Pull request merged**